src/components/atoms/Heading/Heading.tsx

```diff
@@ -1,45 +1,65 @@
 import React, { FC } from 'react';
-import styled from 'styled-components';
+import styled, { css } from 'styled-components';
 import { typography } from '../../../constants/typography';
 import { colors, TColors } from '../../../constants/colors';
-import { maxMedia } from '../../../helpers/responsiveness';
 
-const {
-  sizes: { heading: headingSizes },
-} = typography;
-
-type THeadingSizes = 'h1' | 'h2' | 'h3' | 'h4' | 'h5' | 'h6';
+type THeadingTags = 'h1' | 'h2' | 'h3' | 'h4' | 'h5' | 'h6' | 'span';
 
 interface IStyledHeadingProps {
   /**
    * The HTML5 tag you want to render your heading, it's used to determine the size of the heading as well.
    */
-  as: THeadingSizes | 'span';
+  as: THeadingTags;
   /**
    * Override the default size assigned to the rendered HTML tag.
+   * @default `as`
    */
-  size?: THeadingSizes;
+  size?: keyof typeof typography.sizes.heading;
   /**
    * Whether to add some margin below the rendered heading or not. Applied by default.
+   * @default true
    */
   mb?: boolean;
   /**
    * Accepts a subset of the Zopa brand colors. Same as the ones accepted by `<Text />`.
+   * @default `colors.greyDarkest`
    */
   color?: TColors['white'] | TColors['grey'] | TColors['greyDarkest'];
+  /**
+   * Where the rendered text should be aligned to.
+   * @default 'left'
+   */
+  align?: 'left' | 'right' | 'center';
 }
 
+const {
+  sizes: { heading: headingSizes },
+} = typography;
+
+const lineHeightMap = {
+  display: '76px',
+  h1: '54px',
+  h2: '46px',
+  h3: '36px',
+  h4: '32px',
+  h5: '26px',
+  h6: '20px',
+};
```

Comment:
The reason I inlined the `line-height` on `<Text />`, and `<Heading />`, rather than having them in our `typography` constants is that I don't see much of a use case for them to be there.

It's still useful to keep the `font-size` though in the `typography` constants as they're read in our **form controls components**.

Reply:
Is there any reason why the letter spacing is not recorded?

Reply:
Good catch 👏🏻 , I'll address 👍